Rakefile

```diff
@@ -247,6 +247,11 @@ ensure
   File.delete(output_file) if File.file?(output_file)
 end
 
+desc "Run rhizome (data plane) tests"
+task "rhizome_spec" do
+  sh "COVERAGE=rhizome bundle exec rspec -O /dev/null rhizome"
```

Comment:
In the CI, we usually run rake tasks, so we can change this line to `bundle exec rake rhizome_spec` too

https://github.com/ubicloud/ubicloud/blob/ff5c10bf7e013359b4ff9013458aee1eb6ee07ef/.github/workflows/ci.yml#L116

Reply:
Makes sense, I pushed a commit for that.